rpcs/src/methods/legacy.rs

```diff
@@ -509,55 +524,6 @@ pub enum TransactionStatus<Hash> {
     Invalid,
 }
 
-/// The decoded result returned from calling `system_dryRun` on some extrinsic.
```

Comment:
This stuff requires extra crates/features (eg subct_metadata and all fo the dispatch error stuff) so the logic is left in subxt.